modules/platform/nuxeo-platform-usermanager-core/src/main/java/org/nuxeo/ecm/platform/usermanager/io/NuxeoPrincipalJsonWriter.java

```diff
@@ -111,6 +111,7 @@ public NuxeoPrincipalJsonWriter() {
 
     @Override
     protected void writeEntityBody(NuxeoPrincipal principal, JsonGenerator jg) throws IOException {
+        jg.writeBooleanField("isPartial", principal.isPartial());
```

Comment:
Can you move it to other `is..` fields and keep the same order as the one you use in the `NuxoePrincipal` interface?